### PR TITLE
modified example_tls1.py through MilvusClient

### PR DESCRIPTION
**Pull request by nish112022**
It is to make the example consistent with https://milvus.io/docs/tls.md.If these changes look fine I can modify the other example as well

### CODE REVIEW

**Comment by SimFG** [comment]
@nish112022 thanks your contribution. you need to have a sign for the commit to pass the dco check. try to do:
```
git commit --amend -s
```
and then force to push it

**Comment by XuanYang-cn** [comment]
wait for #2068

**Comment by XuanYang-cn** [comment]
@nish112022 please rebase the latest code to pass the code check actions.
@SimFG Please help review the code

File: examples/example_tls1.py [hunk 1]
```diff
@@ -35,129 +35,89 @@
...
-def create_connection():
+def main():
+    # create a connection
     print(f"\nCreate connection...")
-    connections.connect(host=_HOST, port=_PORT, secure=True, server_pem_path="cert/server.pem", server_name="localhost")
-    print(f"\nList connections:")
-    print(connections.list_connections())
+    milvus_client = MilvusClient(uri=_URI,
+                            secure=True,
+                            server_pem_path="/root/milvus_2.4/milvus/configs/cert/server.pem",
```
Comment (SimFG) [inline examples/example_tls1.py:43]: should use `cert/server.pem`

File: examples/example_tls1.py [hunk 2]
```diff
@@ -35,129 +35,89 @@
...
-    return data[1]
+    print(milvus_client.list_collections())
 
+    # insert 10000 vectors with 128 dimension
+    data_dict = []
+    for i in range(10000):
+        entity = {
+        "id_field": i+1,  # Assuming id_field is the _COLLECTION_NAME of the field corresponding to the ID
+        "float_vector_field": [random.random() for _ in range(_DIM)]
+    }
```
Comment (SimFG) [inline examples/example_tls1.py:72]: please format it

File: examples/example_tls1.py (hunk 1 above)
Comment (nish112022) [reply examples/example_tls1.py]: Fixed it.

**Comment by nish112022** [comment]
@SimFG Added the fix suggested by you. PTAL

**Comment by nish112022** [comment]
@XuanYang-cn SimFg has reviewed my PR.PTAL

**Pull request merged**